sys-boot/plymouth/plymouth-22.02.122-r1.ebuild

```diff
@@ -28,7 +28,7 @@ CDEPEND="
 	drm? ( x11-libs/libdrm )
 	gtk? (
 		dev-libs/glib:2
-		>=x11-libs/gtk+-3.14:3
+		>=x11-libs/gtk+-3.14:3[X]
```

Comment:
New runtime dependencies require a revision bump

Reply:
That makes it an RDEPEND. CDEPEND is just a helper to have things both in RDEPEND and DEPEND, it is not something recognized by the PMS. And if we change an RDEPEND then we have to revbump to propagate those changes to users.

Alternatively, we can also simply not touch the old ebuild and focus on the new one instead.

Reply:
Oh oops! I didn't realize that this was about 22.02.122, I'll bump the revision later, the change in the gtk USE flags was due to a bug here: https://bugs.gentoo.org/850400, would of probably been better to be in its own PR though.